gestionatr/utils.py

```diff
@@ -34,3 +34,13 @@ def get_description(code, table_name):
             )
         )
     return res
+
+
+def validate_xml(data):
+    try:
+        from gestionatr.input.messages import Message
+        m = Message(data)
+        m.parse_xml()
+        return u'Correct File'
+    except Exception, e:
+        return u'WARNING: Invalid File: {0}'.format(str(e.value))
```

Comment:
I think returning an string as 'Correct' or 'Incorrect' is not the best practice here. You can use [namedtuple](https://docs.python.org/2/library/collections.html#collections.namedtuple) to improve the readability an return a tuple which can be accessed by attribute or by position.

Think in something similar to:

```python
ValidationResult = namedtuple('ValidationResult', ['valid', 'error'])
```
Then the method of `validate_xml(data)` something like:

```python
try:
    ...
    return ValidationResult(True, None)
except Exception as e:
    return ValidationResult(False, unicode(e.value))
```

Then when using this method is more *pythonic* to do:

```python
result = validate_xml(xml_data)
if not result.valid:
    print result.error
```
This is the same as

```python
result = validate_xml(xml_data)
if not result[0]:
    print result[1]
```